wsi_superpixel_guided_labeling/web_client/views/vue/components/ActiveLearning/ActiveLearningContainer.vue

```diff
@@ -199,6 +208,12 @@ export default Vue.extend({
                 this.updateMapBoundsForSelection();
                 this.drawLabels();
             });
+            this.viewerWidget.on('g:drawOverlayAnnotation', (element, layer) => {
+                if (element.type === 'pixelmap') {
+                    this.overlayLayer = layer;
+                    updatePixelmapLayerStyle(layer);
+                }
+            });
```

Comment:
I think my previous comment might be due to this. I think that `this.overlayLayer` gets changed to the predictions layer when the Show/hide predictions button is clicked, and then the reference to the labels layer is lost.

Reply:
Yup! I've got a change coming in a minute :slightly_smiling_face: Thanks!